packages/flutter/test/material/data_table_test.dart

```diff
@@ -918,4 +919,115 @@ void main() {
     boxDecoration = tableRow.decoration as BoxDecoration;
     expect(boxDecoration.border.bottom.width, thickness);
   });
+
+  testWidgets('DataTable column heading cell - no sort', (WidgetTester tester) async {
```

Comment:
Can you update this test name? It looks like it is testing for when sort is enabled and disabled.

Reply:
Fixed!